### ISSUE TITLE
dftd3 crash trying to access home directory

### ISSUE THREAD

**Issue by dspoel** [psi4/psi4#1960]
On my HPC cluster compute nodes do not have access to a user's home directory. This makes dftd3 crash with the following error:

RuntimeError: [Errno 13] Permission denied: '/home/s/spoel/dftd3_d6f02c65'

I checked the source code dftd3.f and found
```
      call get_environment_variable("HOME", homedir)
      write (*,*) trim(homedir)
      write(dtmp,'(a,''/.dftd3par.'',a)')trim(homedir),trim(ftmp) 
      inquire(file=dtmp,exist=ex)
```

Do I need to hack the fortran to make this run? I realize this is not strictly a Psi4 problem...

**Comment by hokru** [comment]
Is this the same issue as in #1724? You can try your own workaround or update to the development version.
This has been addressed in PR #1725.

**Comment by dspoel** [comment]
Sorry for being brain dead :(. Indeed it is the same.
